recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.6" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    git_tag: v{{ version }}
```

Comment:
We prefer to use tarballs over cloning from version control. So please change like so. Please fill in the checksum.

```
fn: py_gd-{{ version }}.tar.gz
url: https://github.com/NOAA-ORR-ERD/py_gd/archive/v{{ version }}.tar.gz
sha256: ???
```